components/brave_policy/brave_profile_policy_provider.cc

```diff
@@ -33,6 +40,8 @@ void BraveProfilePolicyProvider::Init(policy::SchemaRegistry* registry) {
   // loading.
   brave_origin_observation_.Observe(
       brave_origin::BraveOriginPolicyManager::GetInstance());
+
+  ad_block_only_mode_policies_loader_->Init();
```

Comment:
Follow pattern of `brave_origin::BraveOriginPolicyManager` instead to not need local state.